Dockerfile

```diff
@@ -9,17 +9,13 @@ COPY . /app
 # Install any needed packages specified in requirements.txt
 RUN pip install --no-cache-dir -r requirements.txt
 
-# Conditionally install django-q2
```

Comment:
Maybe test if you add this, the docker compose command will still work.
Otherwise a profile will be required specifically when running the docker compsoe for simplicity.